3d/rim.py

```diff
@@ -0,0 +1,30 @@
+"""
+This script defines the rim body.
+"""
+from math import cos
+from math import sin
+from math import radians
+
+import cadquery
+from Helpers import show
+
+FDM_MARGIN = 0.1
+
+TIRE_W = 2.5 * 2
+WHEEL_D = 20.5
+WHEEL_D_FDM = WHEEL_D - FDM_MARGIN
+
+REEL_D0 = WHEEL_D_FDM - TIRE_W
+REEL_H0 = 8
+
+PINION_D = 6.70
+PINION_D_FDM = PINION_D + FDM_MARGIN
```

Comment:
I would declare all configurable variables on the top like:

```python
FDM_MARGIN = 0.1
REEL_H0 = 8
TIRE_W = 2.5
WHEEL_D = 20.5
PINION_D = 6.70

OUTER = WHEEL_D - TIRE_W * 2 - FDM_MARGIN
INNER = PINION_D + FDM_MARGIN
```

Reply:
Solve on 70f4e9b22216ff3060ce5249c5d5b3f6b4b410f9

Reply:
Or even... just define inner/outer variables? :innocent: 

We could start using lower case for variables (more Pythonic...)